src/assets/scripts/embed.njk

```diff
@@ -146,7 +172,12 @@ class WebringBanner extends HTMLElement {
           ${this.getStyles()}
           <div class="webring-banner">
             <div class="webring-banner__header">
-              <img class="webring-banner__image" src="${this.image}" alt="" />
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" class="webring-banner__image">
```

Comment:
I'm not 100% happy with this solution since it means one has to adjust this path separately from the .svg if changes are made, but I guess it's not that bad if we get it to look nice with all theme combos.

A possible different solution would be to add an additional parameter which would go into `filter: invert(<param>)`.

Reply:
I don't like the filter option, as it's restricting possibilities quite a lot. Having to embed the svg to make it work is a bit annoying. I'm sure we can do something during the build though, which takes the svg file and embeds it.

Reply:
Makes sense. That can be tackled in another commit though; for now this is great so let's roll with it.